README.md

```diff
@@ -6,6 +6,14 @@ Phlex lets you compose web views in pure Ruby. It’s super-fast, thread-safe an
 
 Documentation can be found at [www.phlex.fun](https://www.phlex.fun).
 
+### Testing
+
+We use [`sus`](https://github.com/ioquatix/sus) execute tests. To run gem's testsuite use the following command
+
+```
+bundle exec sus
+```
+
```

Comment:
This is in `CONTRIBUTING.md`. I wonder if we should just link that from the README under a "Contributing" heading.

Reply:
Ah. My bad missed it.  I just skimmed Readme, and didn't see anything.  :)